examples/caffe2/Dockerfile.example.caffe2

```diff
@@ -0,0 +1,45 @@
+FROM pai.build.base:hadoop2.7.2-cuda8.0-cudnn6-devel-ubuntu16.04
+
+# install deps
+RUN apt-get -y update && \
+    apt-get install -y --no-install-recommends \
+      build-essential \
+      cmake \
+      git \
+      libgoogle-glog-dev \
+      libgtest-dev \
+      libiomp-dev \
+      libleveldb-dev \
+      liblmdb-dev \
+      libopencv-dev \
+      libopenmpi-dev \
+      libsnappy-dev \
+      libprotobuf-dev \
+      libgflags-dev \
+      openmpi-bin \
```

Comment:
Is it possible to use openmpi base image?

Reply:
Is it necessary? I'm not sure whether the environment is consistent.